Privacy.md

```diff
@@ -0,0 +1,31 @@
+#Privacy 
+This is a plain English summary of all of the components within Piwik which may affect your privacy in some way. Please keep in mind that if you use third party Themes or Apps with Piwik, there may be additional things not listed here.
+Each of the items listed in this document can be disabled via Piwik's config.js file. 
+You can read about Piwik privacy policy [here](http://piwik.org/privacy-policy/).
+##Privacy for Piwik admins and website owners
+As admins is the Piwik admin and any other user acessing Piwik
+##disable the automatic update check
+When a new session is started, Piwik check if the current version of Piwik is the latest version of Piwik. If an update is available, a notification appears inside Piwik to let you know.
+This service can be disabled at any time. By setting "Auto update" feature enable_auto_update = 0 " in your configuration file in config/config.ini.php.
+##Cookies
+A cookie is a string of information that a website stores on a visitor’s computer, and that the visitor’s browser provides to the website each time the visitor returns.
+You can disable them in this [way](http://piwik.org/faq/general/faq_157/)
+##Hide Server Url
+By default, the Piwik Javascript code on all tracked websites contains the Piwik server URL. In some cases you might want to hide this Piwik URL completely while still tracking all websites in your Piwik instance. It is possible to do by modifying your Piwik Javascript code, and point it to a “proxy piwik.php script” instead of your Piwik server URL. 
+[Steps] (http://piwik.org/faq/how-to/faq_132/)
+##Access to real time & visitor logs
+As the Piwik administrator, you may decide that giving access to real time & visitor log features are not necessary for your Piwik users. In this case, you can disable the Live plugin in Administration > Plugins.
```

Comment:
> In this case, you can disable the Live plugin in Administration > Plugins.

Suggested: `To keep your user details private, disable the 'Live' plugin in Administration > Plugins:`